ansible_base/jwt_consumer/common/auth.py

```diff
@@ -216,34 +234,29 @@ def get_decryption_key(self, url_or_string: str, ignore_cache: bool = False, val
         cache.set(cache_key, key, timeout=self.get_cache_timeout())
         return key, cached
 
-    def map_user_fields(self, user, token):
+    def map_user_fields(self):
```

Comment:
Do you need protection from this and process(_rbac)_permissions being called before authenticate since they are exposed on the top classes?